src/utils/input-sanitizer.ts

```diff
@@ -0,0 +1,9 @@
+import { escape, trim } from 'validator'
+
+export function sanitizeInputString(inputString: string) {
+    let sanitizedInput = inputString || ''
+    sanitizedInput = trim(sanitizedInput)
+    sanitizedInput = escape(sanitizedInput)
+
```

Comment:
What do you think about repeated white spaces in the middle? E.g. when searching by Postcode those should be valid and give the same results: 'CR0 2GF' , 'CR0(space)(space) 2GF'

Reply:
interesting thought, something that we'll have to look at when we start using the mapping library, it may  handle double spaces elegantly or it may not consider it to be the same postcode.